src/rdiff_backup/selection.py

```diff
@@ -457,6 +459,26 @@ def _filelist_globbing_get_sfs(self, filelist_fp, inc_default, list_name):
             else:
                 yield self._glob_get_sf(line, inc_default)
 
+    def _filesystem_type_get_sf(self, fstype, include):
+        """Return selection function matching filesystem type."""
+        assert include == Select.EXCLUDE or include == Select.INCLUDE, (
+            "Include is {ival}, should be 0 or 1.".format(ival=include))
+        # FIXME Raise an exception on Windows ?
+        # Since it's too costly to get the filesystem type of each file, will identify the mountpoint using devloc
+        psutil = self.rpath.conn.psutil
+        assert psutil, "psutil must be installed"  # FIXME Raise an exception is psutil is not installed ?
+        partitions = self.rpath.conn.psutil.disk_partitions(all=True)
+        devlocs = [self.rpath.conn.os.lstat(p.mountpoint).st_dev for p in partitions if p.fstype == fstype]
```

Comment:
I don't want new direct remote use of Python functions. They need to be wrapped in a new rpath function, which breaks the API 201 BTW (but so does the remote call to psutil). The remote call needs to be marked as API function, it needs to be documented, and the value bumped.

Reply:
If possible, wrap both lines into one remote function, then you'll have only one remote call instead of two (psutil and lstat), this should improve the performance.